<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Часовой пояс        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="TimeZoneComboBox">
       <item>
        <property name="text">
         <string>...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-9:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-3:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+3:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+4:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+5:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+5:45</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+6:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+8:45</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+9:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+10:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+12:45</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTC+14</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Циферблат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="DigitRadioButton">
       <property name="text">
        <string>Цифровой</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="AnalogRadioButton">
       <property name="text">
        <string>Аналоговый</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>300</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>600</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="Label_1">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>* Выберите один из вариантов</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="Label_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>* Выберите один из вариантов</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
